centrifuge/exceptions.py

```diff
@@ -1,55 +1,41 @@
-class CentrifugeException(Exception):
-    """
-    CentrifugeException is a base exception for all other exceptions
+class CentrifugeError(Exception):
+    """CentrifugeError is a base exception for all other exceptions
     in this library.
     """
-    pass
 
 
-class ClientDisconnected(CentrifugeException):
-    """
-    ConnectionClosed raised when underlying websocket connection closed.
-    """
-    pass
+class ClientDisconnectedError(CentrifugeError):
+    """ConnectionClosed raised when underlying websocket connection closed."""
 
 
-class DuplicateSubscription(CentrifugeException):
-    """
-    DuplicateSubscription raised when trying to create a subscription for a channel which
+class DuplicateSubscriptionError(CentrifugeError):
+    """DuplicateSubscriptionError raised when trying to create a subscription for a channel which
     already has subscription in Client's internal subscription registry. Centrifuge/Centrifugo
     server does not allow subscribing on the same channel twice for the same Client.
     """
-    pass
 
 
-class SubscriptionUnsubscribed(CentrifugeException):
-    """
-    SubscriptionUnsubscribedError raised when an error subscribing on channel occurred.
-    """
-    pass
+class SubscriptionUnsubscribedError(CentrifugeError):
+    """SubscriptionUnsubscribedError raised when an error subscribing on a channel occurred."""
 
 
-class Timeout(CentrifugeException):
-    """
-    Timeout raised every time operation times out.
-    """
-    pass
+class ClientTimeoutError(CentrifugeError):
```

Comment:
Let's call it `TimeoutError` as it can be returned from subscription ops too so I think may be a bit confusing

Reply:
TimeoutError is built in exception, we can try something else

Reply:
Probably `OperationTimeoutError`?